pkgs/servers/tt-rss/theme-feedly/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, fetchFromGitHub, ... }: stdenv.mkDerivation rec {
```

Comment:
I don't think you need `...` here. AFAIK the use of `...` is discouraged as well.